packages/graph/README.md

```diff
@@ -1,6 +1,6 @@
-# The HASH Graph API
+# The HASH Graph Query Layer
```

Comment:
@nonparibus Any preference on what we call this and how we refer to it in documentation?

Reply:
@Alfred-Mountfield Grab me in-person to discuss this